docs/AUTOPGKTEST.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Autopkgtest - Defining tests for Debian packages
+category: Documentation for Developers
+layout: default
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+# Test description
+
+Full system integration/acceptance testing is done through [autopkgtests](https://salsa.debian.org/ci-team/autopkgtest/-/blob/master/doc/README.package-tests.rst). These test the actual installed binary distribution packages. They are run in QEMU or containers and thus can do intrusive and destructive things such as installing arbitrary packages, modifying arbitrary files in the system (including grub boot parameters), rebooting, or loading kernel modules.
```

Comment:
We made an agreement to use [semantic line breaks](https://sembr.org/) in new docs. I think here it's particularly pertinent, since we're likely to edit the docs after the import is complete, so it'll be nice to have a diff-friendly formatting.

I guess we don't need fine-grained break, but we should at least not have multiple sentences on the same line. So please add line breaks, at least for individual sentences.